lib/stdlib/src/lists.erl

```diff
@@ -36,7 +36,7 @@
          prefix/2, reverse/1, seq/2, seq/3,
          split/2, sublist/2, sublist/3,
          subtract/2, suffix/2, sum/1,
-         unzip/1, unzip3/1,
+         uniq/1, uniq/2, unzip/1, unzip3/1,
```

Comment:
`uniq/1` should not be mentioned here. It should be in the second `export` in this module (among miscellaneous list functions that don't take funs as arguments).

Reply:
![immagine](https://user-images.githubusercontent.com/852548/158013955-141c631e-85f2-456d-9838-6361bf458950.png)

Reply:
done